api/src/main/java/com/github/streamshub/console/api/support/KafkaContext.java

```diff
@@ -115,4 +158,78 @@ public Optional<String> tokenUrl() {
                     .map(KafkaListenerAuthenticationOAuth.class::cast)
                     .map(KafkaListenerAuthenticationOAuth::getTokenEndpointUri));
     }
+
+    /**
+     * The SchemaRegistryContext contains a per-Kafka registry client
+     * and key/value SerDes classes to be used to handle message browsing.
+     *
+     * The client and SerDes instances will be kept open and reused until
+     * the parent KafkaContext is disposed of at application shutdown.
+     */
+    public class SchemaRegistryContext implements Closeable {
+        private final RegistryClient registryClient;
+        private final MultiformatDeserializer keyDeserializer;
+        private final MultiformatDeserializer valueDeserializer;
+        private final MultiformatSerializer keySerializer;
+        private final MultiformatSerializer valueSerializer;
+
+        SchemaRegistryContext(RegistryClient schemaRegistryClient) {
+            this.registryClient = schemaRegistryClient;
+
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Could this be a constant?

Reply:
Replaced this ObjectMapper with a provided application-scoped mapper.